sqltap/sqltap.py

```diff
@@ -229,13 +257,17 @@ class QueryGroup(object):
     queries, including their query text/time/count, backtrace stacks.
     """
 
+    ParamsID = 1
```

Comment:
what's the rationale here for making this a class-level variable instead of an instance-level var?